test/graphql/business.test.js

```diff
@@ -136,9 +137,11 @@ const setupNock = () => {
       }
     ]
   })
+
+  v1.put('/organisation/organisationId/business-details', orgDetailsUpdatePayload).reply(204)
```

Comment:
I think the plan was to move away from using the fixtures folder? The other mocks in this suite have their payloads inline so it's easier to read the test as a single file.

There's already a tech debt ticket in the backlog to remove these from the remaining places: https://dev.azure.com/defragovuk/DEFRA-FFC/_boards/board/t/DEFRA-FFC%20Customer%20Registry/Stories?text=fix&workitem=560184